<?xml version="1.0"?>
<launch>
  <node name="kuka_control" pkg="tcp_ros" type="kuka_control" output="screen"/>
    <param name="arm_ip" value="172.31.1.147"/>
    <param name="arm_port" value="59152"/>
    <param name="my_ip" value="172.31.1.11"/>
</launch>
